webui/react/src/hooks/useTheme.ts

```diff
@@ -51,13 +52,14 @@ const updateAntDesignTheme = (path: string) => {
  * and storybook Theme decorators and not individual components.
  */
 export const useTheme = (): ThemeHook => {
+  const currentTheme = (() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
   const [ branding, setBranding ] = useState(BrandingType.Determined);
-  const [ mode, setMode ] = useState(() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
-
+  const [ mode, setMode ] = useState(currentTheme);
+  const [ scheme, setScheme ] = useState<DarkLight>(currentTheme);
```

Comment:
**suggestion:** Let's call this `SystemMode` and `setSystemMode`, where it's basically to keep track of what the OS is set to.